ticdc/ticdc-sink-to-kafka.md

```diff
@@ -232,3 +232,21 @@ You can use `partition = "xxx"` to specify a partition dispatcher. It supports f
 > ```
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## Scale out the load of a single large table to multiple TiCDC nodes
+
+This feature splits a single large table into multiple data ranges based on the number of Regions, and distributes these data ranges to multiple TiCDC nodes so that multiple TiCDC nodes can synchronize the large single table at the same time. This feature can solve the following two problems:
```

Comment:
```suggestion
This feature splits a single large table into multiple data ranges based on the number of Regions, and distributes these data ranges to multiple TiCDC nodes so that multiple TiCDC nodes can replicate the large single table at the same time. This feature can solve the following two problems:
```